Casks/r/ringcentral.rb

```diff
@@ -1,5 +1,5 @@
 cask "ringcentral" do
-  arch arm: "-arm"
+  arch arm: "-arm64", intel: ""
```

Comment:
Intel can be omitted as it is automatically an empty string. 

```suggestion
  arch arm: "-arm64"

Reply:
Okay, thanks)